src/cli.ts

```diff
@@ -39,8 +40,31 @@ async function cli(
       await mkdir(dir, { recursive: true });
       await copyFile(src, dest);
     }
+  } else if (action === 'size') {
+    const isSymbolicLink = (m: number) => (m & 61440) === 40960;
+    let bytes = 0;
+    for (const f of allFiles) {
+      const lstat = lstatSync(f);
+      if (isSymbolicLink(lstat.mode)) {
+        bytes += lstat.size;
+      } else {
+        const stat = statSync(f)
+        bytes += stat.size;
+      }
+    }
+    stdout.push(`${bytes} bytes total`)
```

Comment:
It could be preferable to make this KB by default, or choose between bytes or not. Here's an example function - https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable-string/14919494#14919494.

Reply:
We could potentially leverage [`pretty-bytes`](https://www.npmjs.com/package/pretty-bytes) here

Reply:
I started going down that road but I wasn't sure if we wanted to use base 2 (1024) or base 10 (1000) to mean 1KB (I'll need to verify how the API does it) so I left that for a follow up PR

Reply:
Perhaps at least use numeric separators with a standard format like `123,456,789B`?